### ISSUE TITLE
When specify orderBy: TERM_ID there is an error occurred with the cursor

### ISSUE THREAD

**Issue by mohjak** [wp-graphql/wp-graphql#2681]
### Description

When querying list of categories with cursor pagination and order by, then, an error occurred in SQL.

### Steps to reproduce

1. Setup a brand new WordPress website using LocalWP for example
2. Install WPGraphQL plugin and activate it
3. Add 20 sample categories
4. Execute the following query in the GraphiQL IDE
    ```graphql
    query GetCategories($first: Int, $after: String, $last: Int, $before: String) {
      categories(
        first: $first
        after: $after
        last: $last
        before: $before
        where: {order: DESC, orderby: TERM_ID}
      ) {
        pageInfo {
          hasPreviousPage
          hasNextPage
          startCursor
          endCursor
        }
        nodes {
          slug
          databaseId
        }
      }
    }
   {
      "first": 5,
      "after": "YXJyYXljb25uZWN0aW9uOjE0",
      "last": null,
      "before": null
    }
    ```

### Error Log

```sh
[22-Dec-2022 12:41:34 UTC] WordPress database error Column 'term_id' in where clause is ambiguous for query 
			SELECT  t.term_id
			FROM wp_terms AS t  INNER JOIN wp_term_taxonomy AS tt ON t.term_id = tt.term_id
			WHERE tt.taxonomy IN ('category') AND  term_id >= "14" AND ( term_id > "14" OR (  t.term_id < 14 ) ) 
			ORDER BY t.term_id DESC
			 LIMIT 0, 6
		 made by require('wp-blog-header.php'), wp, WP->main, WP->parse_request, do_action_ref_array('parse_request'), WP_Hook->do_action, WP_Hook->apply_filters, WPGraphQL\Router::resolve_http_request, WPGraphQL\Router::process_http_request, WPGraphQL\Request->execute_http, GraphQL\Server\StandardServer->executeRequest, GraphQL\Server\Helper->executeOperation, GraphQL\Server\Helper->promiseToExecuteOperation, GraphQL\GraphQL::promiseToExecute, GraphQL\Executor\Executor::promiseToExecute, GraphQL\Executor\ReferenceExecutor->doExecute, GraphQL\Executor\ReferenceExecutor->executeOperation, GraphQL\Executor\ReferenceExecutor->executeFields, GraphQL\Executor\ReferenceExecutor->resolveField, GraphQL\Executor\ReferenceExecutor->resolveFieldValueOrError, WPGraphQL\Utils\InstrumentSchema::WPGraphQL\Utils\{closure}, WPGraphQL\Type\WPConnectionType->WPGraphQL\Type\{closure}, WPGraphQL\Type\Connection\TermObjects::WPGraphQL\Type\Connection\{closure}, WPGraphQL\Data\DataSource::resolve_term_objects_connection, WPGraphQL\Data\Connection\AbstractConnectionResolver->get_connection, WPGraphQL\Data\Connection\AbstractConnectionResolver->execute_and_get_ids, WPGraphQL\Data\Connection\TermObjectConnectionResolver->get_query, WP_Term_Query->__construct, WP_Term_Query->query, WP_Term_Query->get_terms
```

### Screenshots
![image](https://user-images.githubusercontent.com/299527/209137112-c2714c97-983f-4123-8b45-a05cb0e7e814.png)

### WPGraphQL Version

1.13.7

### WordPress Version

6.1.1

### PHP Version

8.1.9

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have disabled ALL plugins except for WPGraphQL.

- [X] Yes
- [ ] My issue is with compatibility with a specific WordPress plugin, and I have listed all my installed plugins (and version info) above.

**Comment by justlevine** [comment]
Related: #1818

**Comment by jasonbahl** [comment]
This appears to have been fixed between v1.13.7 and 1.16.0 (likely resolved by #2882). 


## v1.13.7


![Image](https://github.com/wp-graphql/wp-graphql/assets/1260765/f6256769-61fd-49d0-a369-4ff82c23aab3)


## 1.16.0

![Image](https://github.com/wp-graphql/wp-graphql/assets/1260765/6a8f2b67-7936-4dae-9cc6-166b4c069384)
